azdev/utilities/path.py

```diff
@@ -87,9 +87,21 @@ def make_dirs(path):
             raise
 
 
+def get_integrated_command_modules():
```

Comment:
I would strongly prefer this logic be applied in `get_path_table` so that the 'mod' collection contains both the independent and integrated modules. Then I would expect the test discovery to "just work".  I originally had methods like "get_command_modules" and "get_extensions" but removed them because I didn't want such an extensive interface. The path table is intended to be the single entry point for all such behavior.

Reply:
That's how I started off implementing this, but in the end I pulled it out because all of the tools that consume get_path_table assume that it is reading from a list of packages, not modules. It seemed like a separate enough flow to me to keep this independent.